src/BuildScriptGenerator/PhpVersions.cs

```diff
@@ -6,12 +6,12 @@ public static class PhpVersions
     {
         public const string PhpRuntimeBaseTag = "20220324.2";
         public const string PhpFpmRuntimeBaseTag = "20220325.1";
-        public const string ComposerVersion = "2.0.8";
+        public const string ComposerVersion = "2.3.4";
```

Comment:
Looks that we make default composer version to be 2.3.4 now and will it be compatible with other php versions?

Reply:
Reverted back to 2.0.8 in the update.